packages/app/config/webpack.prod.js

```diff
@@ -354,7 +354,7 @@ module.exports = merge(commonConfig, {
     new CopyWebpackPlugin([
       {
         from: '../sse-hooks/dist',
-        to: 'public/sse-hooks',
+        to: 'public/[name].[hash].[ext]',
```

Comment:
It should probably be in the sse-hooks folder?

Reply:
added

![CleanShot 2020-11-12 at 15 22 15](https://user-images.githubusercontent.com/1051509/98951692-dc37cb00-24fa-11eb-8b1c-bf7e0d68fd0e.png)

Reply:
I think it should be a single entry probably, the first one.

Reply:
I don't really know why he added this but I would say putting both will still work and we ask when he comes back

Reply:
We wanted to add it to make the url unique for `sse-hooks`, I think if we have the second entry, `sse-hooks` in the file manifest here: https://codesandbox.io/file-manifest.json will maybe still link to the old one (without hash). So removing the second one should make everything work. I believe it should be something like


```
{
  from: '../sse-hooks/dist',
  to: 'public/sse-hooks/[name].[hash].[ext]'
}
```